torch/csrc/distributed/c10d/TCPStore.cpp

```diff
@@ -181,7 +183,11 @@ class SendBuffer {
  public:
   SendBuffer(detail::TCPClient& client, detail::QueryType cmd)
       : client(client) {
-    buffer.reserve(32); // enough for most commands
+    const uint8_t* tmp_ptr = (const uint8_t*)&queryMagicNumber;
+    buffer.reserve(36); // enough for most commands
```

Comment:
Nit: I would do `32 + sizeof(queryMagicNumber)` to make sure future proof.

Reply:
Thanks for the comment, @cbalioglu. I've revised the code.